Cargo.toml

```diff
@@ -6,27 +6,39 @@ description = "A CLI tool built for creating Containerfile templates based on th
 repository = "https://github.com/blue-build/cli"
 license = "Apache-2.0"
 categories = ["command-line-utilities"]
+build = "build.rs"
 
 [dependencies]
 anyhow = "1"
 askama = { version = "0.12.1", features = ["serde-json"] }
 cfg-if = "1.0.0"
 chrono = "0.4"
-clap = { version = "4", features = ["derive"] }
-clap-verbosity-flag = "2.1.1"
+clap = { version = "4", features = ["derive", "cargo", "unicode"] }
+clap-verbosity-flag = "2"
+clap_complete = "4"
 derive_builder = "0.12.0"
+dirs = "5"
 env_logger = "0.10.1"
 futures-util = { version = "0.3.30", optional = true }
 indexmap = { version = "2.1.0", features = ["serde"] }
 log = "0.4"
+nu-ansi-term = "0.50"
```

Comment:
There's a better crate for setting colors on text called [colorized](https://crates.io/crates/colorized) that uses traits so you can just call functions on `&str`s to change the color.